internal/storage/storagetesting/register_bucket_tests.go

```diff
@@ -12,17 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package gcstesting
+package storagetesting
```

Comment:
what are these tests and does these run?

Reply:
This is for fake bucket unit tests. fake/bucket_test.go is calling these tests.

Reply:
Should I move them under fake directory?